tests/e2e/login/privacy-policy.wdio-spec.js

```diff
@@ -0,0 +1,128 @@
+    username: 'online',
+    roles: ['program_officer'],
+    place: {
+      _id: 'hc2',
+      type: 'health_center',
+      name: 'Health Center 2',
+      parent: 'dist1'
+    },
+    contact: {
+      _id: 'fixture:user:onlineuser',
+      name: 'onlineuser'
+    }
+  });
+
+  const parent = placeFactory.place().build({_id:'dist1',type:'district_hospital'});
+
+  before(async () => {
+    await utils.saveDocs([parent, privacyPolicyFactory.privacyPolicy().build()]);
+    await utils.createUsers([ onlineUser, offlineUser]);
+  });
+
+  describe('for an online user', () => {
+    it('should show the correct privacy policy on login', async () => {
+      // After first login, check that privacy policy was prompted to user
+      await loginPage.login(onlineUser.username, onlineUser.password);
+      await browser.waitUntil(async () => {
+        const wrapperText = await (await privacyPolicyPage.privacyWrapper()).getText();
+        return wrapperText.includes(englishTexts.header) && wrapperText.includes(englishTexts.paragraph);
+      }, 'Timed out waiting for English Online Privacy Policy to Display');
+
+      // After accepting, no privacy policy on next load
+      await privacyPolicyPage.acceptPrivacyPolicy();
+      await browser.url('/');
+      expect(await privacyPolicyPage.privacyWrapper()).not.toBeDisplayed();
+
+      // Check display when loading privacy policy page
+      await privacyPolicyPage.goToPrivacyPolicyConfig();
+      await browser.waitUntil(async () => {
+        const wrapperText = await (await privacyPolicyPage.privacyConfig()).getText();
+        return wrapperText.includes(englishTexts.header) && wrapperText.includes(englishTexts.paragraph);
+      }, 'Timed out waiting for english online privacy to display');
+
+      // No privacy policy on 2nd login
+      await browser.reloadSession();
+      await browser.url('/');
+      await loginPage.login(onlineUser.username, onlineUser.password);
+      expect(await privacyPolicyPage.privacyWrapper()).not.toBeDisplayed();
+
+
+      // After login in french, check that privacy policy was prompted to user again
+      await browser.reloadSession();
+      await browser.url('/');
+      await loginPage.login(onlineUser.username, onlineUser.password, 'fr');
+      await browser.waitUntil(async () => {
+        const wrapperText = await (await privacyPolicyPage.privacyWrapper()).getText();
+        return wrapperText.includes(frenchTexts.header) && wrapperText.includes(frenchTexts.paragraph);
+      }, 'Timed out waiting for french online privacy to display');
+      await privacyPolicyPage.acceptPrivacyPolicy();
+    });
+  });
+
+
+  // WDIO currently cannot log in offline users because of service worker race condition
+  // https://github.com/medic/cht-core/issues/7242
+  xdescribe('for a french offline user', () => {
+    let passed = false;
+    afterEach(async () => {
+      if (!passed) {
+        // I suspect this test is failing because of a conflict.
```

Comment:
Because of the change you made to the way you build users, by adding `known: true`, you're actually bypassing what caused this test to fail. Details here: https://github.com/medic/cht-core/issues/6960
Because the user is already known, we never see the tour modal. I know the modal is a pain, but in this instance it did surface this one bug. 
Maybe we could consider not setting `known: true` and handling the modal after login automatically instead?

Reply:
One is enough, but it has to be for an offline user.